ansible/roles/operator-pipeline/templates/openshift/tasks/upload-pipeline-logs.yml

```diff
@@ -36,6 +36,8 @@ spec:
       description: Tekton pipeline run name where logs will be gathered from
   results:
     - name: pipeline_log_url
+  workspaces:
```

Comment:
This task is used in hosted and release pipelines as well. If you want to add a new workspace to it you also need to update all pipelines that use the task. Otherwise, these pipelines will be broken.

Reply:
done